openmetadata-ui/src/main/resources/ui/src/styles/components/button.less

```diff
@@ -75,3 +75,19 @@ button {
   color: @primary-color;
   background: transparent;
 }
+.ant-btn-group {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 12px;
+  .ant-btn {
+    height: 40px;
```

Comment:
not a scalable solution, what will happen if there is a size defined on button group